src/main/java/br/com/caelum/vraptor/util/jpa/extra/ParameterLoader.java

```diff
@@ -69,32 +73,29 @@ public ParameterLoader(EntityManager em, HttpServletRequest request, ParameterNa
 		this.flash = flash;
 	}
 
+	@Accepts
 	public boolean containsLoadAnnotation(ControllerMethod method) {
 		return any(asList(method.getMethod().getParameterAnnotations()), hasAnnotation(Load.class));
 	}
-
-	public void load(@Observes ControllerMethodDiscovered event){
-		
-		ControllerMethod method = event.getControllerMethod();
-		
-		if (!containsLoadAnnotation(method)) return;
-		
+	
+	@AroundCall
```

Comment:
It doesn't always calls `stack.next()`. When the loaded entity is null, in line 94, the interceptor is interrupted after setting `result.notFound();`